### DISCUSSION TITLE
Web page clipper

### DISCUSSION THREAD

**Discussion by iSwiftLiu** [toeverything/AFFiNE#6171] [Ideas]
### 1~3 main use cases of the proposed feature

collection web page

**Comment by tzhangchi** [comment]
this is a fancy feature，we will plan it

**Comment by est7** [comment]
Why turn it off? It's a good feature.people need to make notes on what they clip

**Reply by EYHN** [reply]
Sorry, I closed some inactive issues to keep the team focused.

Since we don't have enough manpower to implement this in a short time, I converted this issue to a discussion where we can discuss it.

**Comment by RichardQian** [comment]
https://chromewebstore.google.com/detail/affine-web-clipper/mpbbkmbdpleomiogkbkkpfoljjpahmoi
It's released last week, if you are still interested, you are welcome to give it a try

**Comment by bytersproblem** [comment]
It does not open self-hosted instances.

**Reply by RichardQian** [reply]
did you added your server into the extension? and did you upgrade your instance to the latest version? worked for me

https://github.com/user-attachments/assets/3dd3367c-4460-4cd7-ab89-97050aafdbd2

**Reply by iSwiftLiu** [reply]
cool!

**Comment by bytersproblem** [comment]
Could my problems be related to nginx setup?
Thanks

**Comment by uknth** [comment]
Thank you, the extension published on chrome-webstore works for me. However, it doesn't actually clip the URL of the page. Usually, the web-clippers do both, convert the content to markdown & add a property called "source" which also has the URL of the page. 

Is there a way to achieve this?

**Reply by RichardQian** [reply]
noted, yes we did misss that, will see what we can do

**Reply by RichardQian** [reply]
added "clipped from this url" at the end of the content, as a quick mitigation of this problem

**Comment by df911** [comment]
Hello,

I use  Affine web clipper but the images are not pasted into the note. 
Is it a limitation ?

**Reply by RichardQian** [reply]
not intentionally, images successfully parsed will be sent to AFFiNE as a blob attachment, but if the parsing went wrong, that image might be missed.  You may try this sample: https://www.economist.com/weeklyedition/

**Reply by df911** [reply]
thanks for information.
I don't have image if I clip this page

**Reply by RichardQian** [reply]
rather odd, this is the snapshot in my case, chrome browser and latest extension version(added "clipped from this url") 
[fb32b040-b539-4d91-8bb4-6dc1daf27d8f.bs.zip](https://github.com/user-attachments/files/21544961/fb32b040-b539-4d91-8bb4-6dc1daf27d8f.bs.zip)

**Reply by df911** [reply]
I ran several tests:
- with my hosted version, I don't have the images
- with the Cloud version, I do have the images.

What could be causing the error?

**Reply by RichardQian** [reply]
Not quite sure at this stage, maybe WebSocket or https support so to avoid certain fallback?

**Reply by CX330Blake** [reply]
Yeah in some pages, I cannot get the images but some pages can. For example, I can't get the image on https://cloud.google.com/blog/topics/threat-intelligence/gemini-for-malware-analysis this site
